inst/NEWS.Rd

```diff
@@ -3,8 +3,14 @@
 \newcommand{\cpkg}{\href{http://CRAN.R-project.org/package=#1}{\pkg{#1}}}
 \section{Changes in version 2.8-4 (GitHub-only-release date:?-?-2022)}{
   \itemize{
-    \item Part 5: dayborder != 0 is now handled in g.part5.wakesleepwindow
-    \item Part 5: excludefirstlast.part5 is now applied per window type in g.report.part5
+    \item Minor fix to g.getbout because it included one epoch beyond the period of interest in the identified bout.
```

Comment:
Clarify that this was only the cased for epoch lengths below 30 seconds and when boutcriter was not 1.

Reply:
Resolved in [PR #716](https://github.com/wadpac/GGIR/pull/716) and removed this item from this branch.